.evergreen/run-mongodb-aws-test.sh

```diff
@@ -39,37 +39,12 @@ fi
 # show test output
 set -x
 
-# Workaround macOS python 3.9 incompatibility with system virtualenv.
-if [ "$(uname -s)" = "Darwin" ]; then
-    VIRTUALENV="/Library/Frameworks/Python.framework/Versions/3.9/bin/python3 -m virtualenv"
-else
-    VIRTUALENV=$(command -v virtualenv)
-fi
-
-authtest () {
-    if [ "Windows_NT" = "$OS" ]; then
-      PYTHON=$(cygpath -m $PYTHON)
-    fi
-
-    echo "Running MONGODB-AWS authentication tests with $PYTHON"
-    $PYTHON --version
-
-    $VIRTUALENV -p $PYTHON --never-download venvaws
-    if [ "Windows_NT" = "$OS" ]; then
-      . venvaws/Scripts/activate
-    else
-      . venvaws/bin/activate
-    fi
-    python -m pip install '.[aws]'
-    python test/auth_aws/test_auth_aws.py -v
-    deactivate
-    rm -rf venvaws
-}
-
-PYTHON=${PYTHON_BINARY:-}
-if [ -z "$PYTHON" ]; then
+if [ -z "$PYTHON_BINARY" ]; then
```

Comment:
Do we need this file at all? What if we put the new contents directly into `config.yml`?

Reply:
We would still run these two files as entrypoints for local testing.  I wanted to keep the logic out of config.yml where possible, and there are parts of this that we don't want to echo and parts that we do.

Reply:
Makes sense!